src/Context/DebugScope.php

```diff
@@ -49,18 +49,18 @@ public function detach(): int
         if (($flags & ScopeInterface::DETACHED) !== 0) {
             trigger_error(sprintf(
                 'Scope: unexpected call to Scope::detach() for scope #%d, scope was already detached %s',
-                spl_object_id($this),
+                spl_object_id($this->scope),
```

Comment:
I think using `$this` would be more correct, cause as API user you see `DebugScope` as actual scope.

If you got some bug and start logging `spl_object_id($scope)`, you will get different id's and wold not be able to get sence of what's going on.

Reply:
yes, that makes sense. Thanks for the feedback, updated.